<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resShow">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>0</y>
      <width>320</width>
      <height>64</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>511</width>
      <height>64</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 2px solid white;
	border-radius: 5px
}
font: 900 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerYBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="power2Btn">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percentBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrtBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>135</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>135</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>135</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn8">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>135</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtractBtn">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>135</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cbrtBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>cbrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplyBtn">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn5">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>265</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divideBtn">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>265</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn2">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>265</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>265</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="factorBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>265</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>x!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negateBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>330</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalBtn">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>330</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>129</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dotBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 3px;
}
QPushButton:hover {
	background-color: rgb(47, 47, 47);
	border-radius: 6px;
}

QPushButton:pressed {
	background-color: rgb(72, 72, 72);
	border-radius: 9px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
